container/src/services/container.service.ts

```diff
@@ -69,122 +69,126 @@ export class ContainerService {
-                  data: event.data.data,
-                  keepBrowserHistory: event.data.keepBrowserHistory
-                });
-                break;
-              case LuigiInternalMessageID.SHOW_CONFIRMATION_MODAL_REQUEST:
-                this.dispatch(Events.SHOW_CONFIRMATION_MODAL_REQUEST, targetCnt, event.data.data);
-                break;
-              case LuigiInternalMessageID.SHOW_LOADING_INDICATOR_REQUEST:
-                this.dispatch(Events.SHOW_LOADING_INDICATOR_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.HIDE_LOADING_INDICATOR_REQUEST:
-                this.dispatch(Events.HIDE_LOADING_INDICATOR_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.SET_CURRENT_LOCALE_REQUEST:
-                this.dispatch(Events.SET_CURRENT_LOCALE_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.LOCAL_STORAGE_SET_REQUEST:
-                this.dispatch(Events.LOCAL_STORAGE_SET_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.RUNTIME_ERROR_HANDLING_REQUEST:
-                this.dispatch(Events.RUNTIME_ERROR_HANDLING_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.SET_ANCHOR_LINK_REQUEST:
-                this.dispatch(Events.SET_ANCHOR_LINK_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.SET_THIRD_PARTY_COOKIES_REQUEST:
-                this.dispatch(Events.SET_THIRD_PARTY_COOKIES_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.BACK_NAVIGATION_REQUEST:
-                this.dispatch(Events.BACK_NAVIGATION_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.GET_CURRENT_ROUTE_REQUEST:
-                this.dispatch(Events.GET_CURRENT_ROUTE_REQUEST, targetCnt, event);
-                break;
-              // TODO: discuss if actually needed as the only scenario is when microfrontend initially starts
-              case LuigiInternalMessageID.NAVIGATION_COMPLETED_REPORT:
-                this.dispatch(Events.NAVIGATION_COMPLETED_REPORT, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.UPDATE_MODAL_PATH_DATA_REQUEST:
-                this.dispatch(Events.UPDATE_MODAL_PATH_DATA_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.CHECK_PATH_EXISTS_REQUEST:
-                this.dispatch(Events.CHECK_PATH_EXISTS_REQUEST, targetCnt, event);
-                break;
-              case LuigiInternalMessageID.SET_DIRTY_STATUS_REQUEST:
-                this.dispatch(Events.SET_DIRTY_STATUS_REQUEST, targetCnt, event);
-                break;
-              default:
-                console.warn('Functionality not yet implemented: ', msg);

```

Comment:
I would expect only this **console.warn** here to be affected/removed in this PR to solve the referenced issue. 

Refactoring is good and **needed** but better leave it to another Issue/PR as this area of code here is very often used and we want to avoid accidentally introducing a breaking change or bug. This needs to be throughly tested that it works the same and that would take some time. For such refactoring I would leave it as is for now.